src/main/java/me/clip/placeholderapi/expansion/manager/LocalExpansionManager.java

```diff
@@ -308,6 +315,19 @@ public boolean register(@NotNull final PlaceholderExpansion expansion) {
     return true;
   }
 
+  /**
+   * Overload for {@link #register(PlaceholderExpansion, boolean)} to provide backwards compatibility for expansions / plugins
+   * that call this method directly. It is the equivalent of {@code register(expansion, false)}
+   * @param expansion the expansion to register
+   * @return if the expansion was registered
+   * @deprecated use {@link #register(PlaceholderExpansion, boolean)} directly
+   */
+  @Deprecated
```

Comment:
Do we actually need this? Since it's marked as Internal API usage is it imo not really necessary to add docs and also deprecate what is supposed to be only used by us.

Reply:
I mean, there's nothing wrong with documenting and deprecating internal use methods.